bin/mainflux-docker.sh

```diff
@@ -81,13 +81,13 @@ _start() {
   printf "\nWaiting for Cassandra to start. This takes time, please be patient...\n"
 
   # Wait until Cassandra is ready to accept cqlsh commands
-  # or timeout after 15 sec
+  # or timeout after 100 sec
```

Comment:
Timeout of 100 sec looks too much to me. Would 30 sec do for your machine? Please find a minimal TO.